src/util/token.js

```diff
@@ -23,6 +15,27 @@ const token = (seed, publicKey) => {
   )
 }
 
+const authTokenFromSharedSecret = (sharedSecretBuffer) => {
+  return base64url(
+    crypto.createHmac('sha256', sharedSecretBuffer)
+      .update(AUTH_HMAC_INPUT, 'ascii')
+      .digest()
+  )
+}
```

Comment:
can you merge these two functions into one? https://github.com/interledgerjs/ilp-plugin-virtual/blob/15f806b549dbfe246adac580ab811f7ce0dff1ec/src/util/token.js#L8-L24